packages/angular_devkit/build_angular/src/deploy/index.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { createBuilder } from "@angular-devkit/architect";
+
+const NotImplemented = `
+
+Cannot find "deploy" target for the specified project.
+
+You should add a package that implements deployment capabilities for your
+favorite platform.
```

Comment:
Is it intentional that this is in another new line?

Reply:
Yep